data-connector-lib/src/dpk_connector/core/spiders/sitemap.py

```diff
@@ -88,11 +90,15 @@ def __init__(
                     self.focus_paths.add(path)
 
         # Domains and mime types filtering
-        self.allowed_domains = set(
-            allow_domains
-            if len(allow_domains) > 0
-            else [get_etld1(url) for url in seed_urls]
```

Comment:
Let's say the URL is : http://www.research.ibm.com and domain focus is set to true. Will this restrict to *.research.ibm.com or will it restrict to *.ibm.com ?

Reply:
Note that the parameter I introduced this time is "sub"domain_focus. The crawler focuses the input domain by default: the seed url: https://www.research.ibm.com/ -> the crawl will be restricted to *.ibm.com
If `subdomain_focus` is set to true, the crawl will be restricted to *.www.research.ibm.com.

Reply:
Nice! Thanks for the clarification @hmtbr -san